src/snowflake/snowpark/_internal/xml_reader.py

```diff
@@ -52,6 +52,36 @@ def get_file_size(filename: str) -> Optional[int]:
         return file_obj.tell()
 
 
+def tag_is_self_closing(
+    file_obj: Union[BinaryIO, SnowflakeFile],
+    chunk_size: int = DEFAULT_CHUNK_SIZE,
+) -> Tuple[bool, int]:
+    """
+    Return ``(is_self_closing, end_pos)`` for the opening tag at `start_pos`.
+    ``end_pos`` is the byte offset one past the terminating ``>`` of that
```

Comment:
from the description it seems that the function should also take `start_pos` as an input arg but that is not the case. Does the docstring need to be updated?

Reply:
good catch, fixed